src/pages/forget-password/forms/ResetPassForm.js

```diff
@@ -0,0 +1,93 @@
+import React, { useContext, useState } from 'react'
+import { Grid, IconButton, InputAdornment } from '@mui/material'
+import CustomTextField from 'src/components/Inputs/CustomTextField'
+import { Grow } from 'src/components/Shared/Layouts/Grow'
+import { VertLayout } from 'src/components/Shared/Layouts/VertLayout'
+import { useForm } from 'src/hooks/form'
+import toast from 'react-hot-toast'
+import * as yup from 'yup'
+import { AuthContext } from 'src/providers/AuthContext'
+import { useAuth } from 'src/hooks/useAuth'
+import { useError } from 'src/error'
+import NewPassword from 'src/components/Shared/NewPassword'
+import axios from 'axios'
+import { useRouter } from 'next/router'
+
+const ResetPassForm = ({ labels, username = '' }) => {
+  const [score, setScore] = useState(0)
```

Comment:
Why are we declaring the state here if it's only used in the NewPassword ? Wouldn't it be better to declare it inside the NewPassword component itself?

Reply:
this is a concept done already by rebecca in newpassword 
and i followed it.. 
i guess in the future modifications will be done 
but this is the initial version for now